builds/misc/images-release.yaml

```diff
@@ -19,7 +19,6 @@ jobs:
             docker login '$(build.registry.address)' --username '$(build.registry.user)' --password '$(build.registry.password)'
             docker login '$(registry.address)' --username '$(registry.user)' --password '$(registry.password)'
         displayName: 'Docker Login'
-      - template: ../templates/install-dotnet2.yaml
```

Comment:
Yay!

Reply:
There are other YAML files that use this template too:
- builds/checkin/dotnet.yaml
- builds/ci/dotnet.yaml
- builds/misc/images.yaml

Should we remove it from these as well (and remove the template)?

Reply:
tests are failing to run after I removed dotnet2 install, I am still trying to figure out why